next-level/skills/team-execute/SKILL.md

```
@@ -0,0 +1,137 @@
+Task tool with:
+  subagent_type: "checkpoint-reviewer"
+  team_name: "epic-<name>"
+  name: "reviewer"
+  model: "sonnet"
+  mode: "default"
+  prompt: <checkpoint context — plan, completed task, test results, git diff>
+```
+
+## Phase 3: Execute in Waves
+
+### Wave 1: Independent Tasks
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)